standards/countries.rst

```diff
@@ -0,0 +1,44 @@
+.. _countries:
+
+A common list of countries
+==========================
+
+Having a standardized list of country names is an important prerequisite for region
+definitions, scenario analysis and model comparison.
+
+The :class:`nomenclature` package (`read the docs`_) builds on the :class:`pycountry`
+package to provide a standardized list if country names based on the
+`ISO 3166-1 standard`_.
+
+For consistency with established conventions in the modelling community, several country
+names are shortened compared to ISO 3166-1, e.g. from "Bolivia, Plurinational State of"
+to "Bolivia". Also, "Kosovo" is added (with *alpha_3* code "KOS", following the
```

Comment:
I would put a line break here to make it clear that this is different to changing the name of a country:
```suggestion
to "Bolivia".
Also, "Kosovo" is added (with *alpha_3* code "KOS", following the
```